data/interfaces.txt

```diff
@@ -825,6 +824,12 @@ interface "map buttons" bottom right
 	button i "M_issions"
 		from -170 -50 to -80 0
 	
+	active
+	sprite "ui/dialog cancel"
+		from -170 -90 to -80 -40
+	button f "_Find..."
+		from -170 -90 to -80 -40
```

Comment:
Having so many buttons in the bottom right corner that they have to be arranged in a grid two rows high is rather awful UI design. The +/- buttons are already bad enough, but adding this makes the UI look even more cluttered.

Reply:
I wanted to go for the lower left corner, but it overlaps the pointer panel when viewing missions.
My second choice was above the +/- buttons, but we'd need to add a rotation keyword to the interface sprite specification (or add a a rotated version of ui/dialog cancel sprite, which seemed like too much) to get the sprite decorations between the +/- and the dialog sprite to play nicely.